index.html

```diff
@@ -8,11 +8,11 @@
   </head>
   <body>
     <header>
-      <h2 id="page-title">INTENTION TIMER</h2>
+      <h2 id="pageTitle">INTENTION TIMER</h2>
```

Comment:
Thanks for taking the time to make all these small changes to the class and id attributes